shepherd/RecipeManager.py

```diff
@@ -0,0 +1,77 @@
+from Utils import *
+
+
+class RecipeManager():
+
+    master_recipes = [RECIPE_PROBABILITIES.EASY_RECIPE, RECIPE_PROBABILITIES.MEDIUM_RECIPE, \
+                      RECIPE_PROBABILITIES.HARD_RECIPE, RECIPE_PROBABILITIES.VERY_HARD_RECIPE]
+    gold_recipes = []
+    blue_recipes = []
+    gold_recipes_completed = 0
+    blue_recipes_completed = 0
+
+    def generate_recipes(self):
```

Comment:
self should not be an argument when the class is not going to be instantiated.

Reply:
How can we access the class variables above then?